### PR TITLE
[stable/Dex] Fixed hard coded servicePort in ingress template

### LINKED ISSUE #12727: [stable/Dex] Ingress template has the servicePort hard coded to 8080
**Is this a request for help?**:

No.

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

Bug.

**Version of Helm and Kubernetes**:

2.13.0
1.11.5

**Which chart**:

Dex

**What happened**:

The ingress template has the service port hard coded to 8080.

**What you expected to happen**:

Use the service port specified in the values.

**How to reproduce it** (as minimally and precisely as possible):

Specify a service port in the values other than 8080.

**Anything else we need to know**:

PR is on the way.

### PR DESCRIPTION
**Pull request by michaelajr**
#### What this PR does / why we need it:

@kfox1111 @sstarcher @rendhalver

Fixes hard coded servicePort in the ingress template.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

fixes #12727 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/dex/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: dex
-version: 0.8.0
+version: 0.8.1
```
Comment (cpanato) [inline stable/dex/Chart.yaml]: @michaelajr lets bump to 1.0.0 since this chart is in the stable folder
```suggestion
version: 1.0.0
```

**Comment by michaelajr** [comment]
@cpanato Done.

**Comment by michaelajr** [comment]
Did a pull and merge to bring this PR up to date with master. Hoping for a merge soon. Thanks.

**Pull request merged**